christian-galarza/lesson-1/car_app/src/main/Main.java

```diff
@@ -0,0 +1,39 @@
+package main;
+ public class Main {
+    public static void main(String[] args) {
+        Car car = new Car();
+        double distance = car.carDistance();
+         //create the car with tank capacity and performance.
+        Car chariot = new Car(0,0);
+        Car f1mercedes = new Car(100, 0.1);
+        Car f1ferrari = new Car(100, 0.15);
+        Car f1mclaren = new Car(105, 0.23);
+        Car nissanterrano = new Car(300,0.9);
+        //insert the quality of gas and power on the car.
+         f1mercedes.fillGas(100);
```

Comment:
As part of good practices we have to keep the indentation clear.
Please fix the indentation in all part that you see is not correct